web/controllers/stripe_connect_account_controller.ex

```diff
@@ -3,15 +3,15 @@ defmodule CodeCorps.StripeConnectAccountController do
   use JaResource
 
   alias CodeCorps.StripeConnectAccount
-  alias CodeCorps.StripeService.StripeConnectAccountService
+  alias CodeCorps.StripeService.StripeManagedConnectAccountService
```

Comment:
Would just keep this `StripeConnectAccountService` personally. `managed` is just a param.